README.md

```diff
@@ -14,51 +14,30 @@ _Collaborate and work together on GitHub._
 </header>
 
 <!--
-  <<< Author notes: Course start >>>
-  Include start button, a note about Actions minutes,
-  and tell the learner why they should take the course.
+  <<< Author notes: Step 4 >>>
+  Start this step by acknowledging the previous step.
+  Define terms and link to docs.github.com.
 -->
 
-## Welcome
+## Step 4: Suggest changes
 
-All great projects start with collaboration. Pull requests are the foundation of teamwork on GitHub — and pull request reviews give you the ability to work together and discuss changes specific to a pull request by commenting, requesting changes, or approving.
+_Nice work reviewing that pull request :sparkles:_
 
-- **Who is this for**: Developers, new GitHub users, users new to Git, students, managers, teams.
```

Comment:
i am writing a comment. the purpose of this comment is to send hate to the creator of this branch. in all my years of life, never have i ever seen a repository so bland as this one. a great man once said that the first step to doing anything is taking that first step. i guess he forgot to account for the creation of this repository. the owner of this repository taking the first step to create this repository may well have doomed humanity. with the contents of this repository being so utterly poo poo doo doo, it must have backdated humanity many a years. i must say that i wish to have never been born for seeing this repository get created has lit a candle in me that i never knew was there. i want to burn down my house. i want to burn down my brother's house. i want to burn down my sister's house. i understand that we are the branches that come from the same family tree but i want to burn down the family tree. in fact, i wish i was the family tree so i could burn myself down. in the end, i wish more than anybody to be a burning bush.